test/mock_modbus_server.py

```diff
@@ -14,7 +14,7 @@
 
 from pymodbus.device import ModbusDeviceIdentification
```

Comment:
This line fails. Try `from pymodbus.pdu.device import ModbusDeviceIdentification`

Reply:
changed to from pymodbus.pdu.device import ModbusDeviceIdentification